data/languages/fr60.sinc

```diff
@@ -506,15 +684,15 @@ DIR8B_REL: reloc is dir8 [ reloc = dir8 * 1; ] {
 
 # ====
 
-:LDI_32 #i32, ri48_4 is op48_12=0x9f8 & i32 & ri48_4 {
+:LDI^":32" #i32, ri48_4 is op48_12=0x9f8 & i32 & ri48_4 {
```

Comment:
Thanks for introducing me to `^`. Helps keeps things aligned with the datasheets.